codeforces/src/views/api.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint, jsonify
+
+import settings
+from codeforces.src.database.data_base import DbClient
+
+
+bp = Blueprint("api", __name__, url_prefix="/codeforces/api")
+db_client = DbClient(url=settings.cities_path)
+
+
+@bp.get("/students")
+def get_all_students_json():
+    students_db = db_client.students
+    students = [
+        student.dict() for student in students_db
+    ]
+
+    return jsonify(students)
```

Comment:
You are using **students** variable only in this place, as well as **students_db**.
So, maybe it will be better to use something like this (also we can use **generator** here)
```
  return jsonify(student.dict() for student in db_client.students)
```

Reply:
```python
  return jsonify([student.dict() for student in db_client.students])
```